web/xmds.php

```diff
@@ -405,8 +405,12 @@
     }
 
     // logProcessor
+    // Get the default time zone based on the CMS settings and add a fallback value
+    $defaultTimezone = $container->get('configService')->getSetting('defaultTimezone') ?? 'UTC';
```

Comment:
The `setTimezone` call is right, but I think this is in the wrong place (this will only work for XMDS).

Should we do this everywhere that we call `new Logger(` before then adding that to the DI container?